src/Project/functions/copyInclude.ts

```diff
@@ -9,8 +10,19 @@ export function copyInclude(data: ProjectData) {
 		data.projectOptions.type !== ProjectType.Package &&
 		!(data.projectOptions.type === undefined && data.isPackage)
 	) {
-		benchmarkIfVerbose("copy include files", () =>
-			fs.copySync(INCLUDE_PATH, data.projectOptions.includePath, { dereference: true }),
-		);
+		benchmarkIfVerbose("copy include files", () => {
+			for (const fileName of fs.readdirSync(INCLUDE_PATH)) {
+				const fromPath = path.join(INCLUDE_PATH, fileName);
+				const toPathLuau = path.join(data.projectOptions.includePath, fileName);
+				const toPathLua = toPathLuau.replace(/\.luau$/, ".lua");
+				if (data.projectOptions.luau) {
+					if (fs.existsSync(toPathLua)) fs.unlinkSync(toPathLua);
+					fs.copySync(fromPath, toPathLuau, { dereference: true });
+				} else {
+					if (fs.existsSync(toPathLuau)) fs.unlinkSync(toPathLuau);
+					fs.copySync(fromPath, toPathLua, { dereference: true });
+				}
+			}
```

Comment:
I think it makes more sense to clear the `include` directory instead of checking the specific files.
Hard-coding the `.luau` conversion is ugly. Ideally this should be done via PathTranslator.

Reply:
> I think it makes more sense to clear the `include` directory instead of checking the specific files. 

I considered this, but I am concerned about someone accidentally doing `--includePath .` and then we destroy their whole project. Hopefully, they'd use git but.. yeah.

> Hard-coding the `.luau` conversion is ugly. Ideally this should be done via PathTranslator.

I think this is a bit overkill. PathTranslator is complex because it's mapping src <-> out.